tools/lint-hotkeys

```diff
@@ -67,16 +67,16 @@ def lint_hotkeys_file() -> None:
             print(
                 f"Duplicate key combination for keys {duplicate_keys} for category ({HELP_CATEGORIES[action]}) detected"
             )
-            error_flag = 1
-    if error_flag == 1:
+            error_flag = True
+    if error_flag is True:
```

Comment:
nitpick:
```suggestion
    if error_flag:
```

Reply:
Thanks for the catch! :melting_face:
Updated.